lib/python/qmk/cli/lint.py

```diff
@@ -18,7 +18,7 @@
 def _list_defaultish_keymaps(kb):
     """Return default like keymaps for a given keyboard
     """
-    defaultish = ['ansi', 'iso', 'via']
+    defaultish = ['ansi', 'iso', 'via', 'vial']
```

Comment:
```suggestion
    defaultish = [
    	'ansi',
    	'iso',
    	'via',
    	'vial' # This is only here to flag it as "testable", so it doesn't fly under the radar during PRs
    	]
```
...or something to that effect?